README.md

```diff
@@ -8,31 +8,52 @@ A GAP package for group recognition
 ## Installation
 
 To get the newest version of this GAP 4 package download the
-archive file `recog-x.x.tar.gz` and unpack it using
+archive file `recog-x.x.tar.gz` from
+[here](https://github.com/gap-packages/recog/releases/latest).
+
+### Standard installation
+Unpack the archive using
 
     tar xvf recog-x.x.tar.gz
 
-Do this in a directory called `pkg`, preferably (but not necessarily)
-in the `pkg` subdirectory of your GAP 4 installation. It creates a
+Do this in the `pkg` subdirectory of your GAP 4 installation. It creates a
 subdirectory called `recog`.
 
-This is all which is needed if you installed the package in the standard
-`pkg` subdirectory.
+### Custom installation
+
+If you want to contribute to recog you might want to install it
+into a custom directory. For example into the directory `~/packages`. Then you
+can clone recog (or perhaps your personal fork of it) into `~/packages` via
+
+    cd ~/packages
+    git clone git@github.com:gap-packages/recog.git
+
+To enable GAP to find recog you can start GAP via
+
+    gap --packagedirs "~/packages;"
```

Comment:
Note that this is a very new option, IIIRC added in GAP 4.14 (but we should check GAP's `CHANGES.md`).

So I think this should be listed as a warning.

let's talk about this later

Reply:
I've checked and it was introduced in GAP 4.15 in September. This is the respective [PR](https://github.com/gap-system/gap/pull/5873).